css/symbiota/main.css

```diff
@@ -263,6 +264,10 @@ ul {
   padding-inline-start: 40px;
 }
 
+ul li {
```

Comment:
This will impact styling of all ul list items across the portal.  Have you checked other pages that contain unordered lists to confirm this is desired?

Reply:
Loaded into dev002 - generally looks good.  One probable unintended impact is an increase in the bottom margin of the options in the header pulldown menu.

Original:
![image](https://github.com/user-attachments/assets/e730adc3-53a2-4d21-9331-f1a2d0f26112)

Proposed:
![image](https://github.com/user-attachments/assets/cfec33c3-bdae-4b75-8baf-da1ab8a49a56)

Reply:
If this is not desired I would recommend making the selector more specific - perhaps all ul li elements within the main block?
ex. main ul li {

or

adjusting the menu specific css to compensate.

Reply:
I did notice that but wasn't sure how impactful it was. I think it would likely be preferred to _not_ have the extra spacing in the menu, so I think it does make sense to adjust accordingly. Do you think option A or option B of yours (above) would be the best approach?

Reply:
I would go with option A.